res/xml/qwertz.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?> 
+<keyboard>
+  <row>
+    <key key0="q" key1="esc" key2="1" key3="\@" key4="!"/>
+    <key key0="w" key1="^" key2="2" key3="&quot;" key4="°"/>
+    <key key0="e" key1="§" key2="3" key4="€"/>
+    <key key0="r" key2="4" key3="$"/>
+    <key key0="t" key2="5" key3="%"/>
+    <key key0="z" key2="6" key3="&amp;" key4="{"/>
+    <key key0="u" key2="7" key3="ü" key4="}"/>
+    <key key0="i" key1="(" key2="8" key4="["/>
+    <key key0="o" key1=")" key2="9" key3="ö" key4="]"/>
+    <key key0="p" key1="=" key2="0" key3="~" key2="\?"/>
```

Comment:
`~` is also on the second row.

Reply:
Also the `?` can't be `key2`:
```suggestion
    <key key0="p" key1="=" key2="0" key4="\?"/>
```

Reply:
ad 1.) Yes, that's one of the duplicates that could be removed – I didn't know about ~ + space key. On a German keyboard, ~ is AltGr +  plus key. Quick one hand typing!

ad 2.) Correct, will fix!

Reply:
The `~` on the second row really is the ASCII character `~`. It's not `accent_tilde`, which will be hidden for you most probably.